Campaigns/apt sofacy.txt

```diff
@@ -0,0 +1,7 @@
+// Original Sigma Rule: https://github.com/Neo23x0/sigma/blob/master/rules/apt/apt_sofacy.yml
+// Questions via Twitter: @janvonkirchheim
+ProcessCreationEvents
+| where EventTime > ago(7d)
+| where ProcessCommandLine matches regex @'rundll32.exe %APPDATA%\.*\.dat\",.*'
```

Comment:
Regarding `%APPDATA%\.*`
it could be that regex engine will interpret that as `\.` = which means a real dot. To work around this wrap the dot with `[ ]` like this: `%APPDATA%\[.]*\`